integration/default_set.yaml

```diff
@@ -276,3 +276,10 @@ tests:
         input: data/zeek/conn.log.gz
       - command: -N export -n 10 arrow '#type == "zeek.conn"'
         transformation: python @./misc/scripts/print-arrow.py
+  Type Registry:
+    tags: [import]
+    steps:
+      - command: -N import -n 1 test
+      - command: -N status
+        transformation: jq '.node."type-registry"."type-registry"'
```

Comment:
I don't quite like that both keys are named `type-registry`. Maybe I should rename it to statistics?